docs/layers.md

```diff
@@ -507,8 +507,6 @@ _TIP: Some `landuse` features only exist as point features in OpenStreetMap. Fin
 * `military`
 * `national_park`
 * `nature_reserve`
-* `natural_forest` - _See planned bug fix in [#1096](https://github.com/tilezen/vector-datasource/issues/1096)._
```

Comment:
Please revert these changes for `natural_forest` and `natural_park` throughout this PR, and add the leaf type kind_detail to both.

Reply:
Moved to #1469